services/121-service/src/payments/payments.service.ts

```diff
@@ -272,7 +272,6 @@ export class PaymentsService {
       totalMultiplierSum =
         totalMultiplierSum + registration.paymentAmountMultiplier;
       if (
-        !dryRun && // This is only needed in actual doPayment call
```

Comment:
Is this because the `dryRun`-flag will be no longer explicitly `false` because of some stricter typing/interface-reuse?

Or is it a 'safety-flag' the should've only be removed during development?

Reply:
No, it's because we are now using the `fspsInPayment` in the payment summary, so we also need to get that information when it is a dry run.

Reply:
Aha. That endpoint is called in an other-way-around-way then I thought then.